tests/e2e/solidity/suites.go

```diff
@@ -42,6 +41,10 @@ func stopSubnet() {
 
 var _ = utils.DescribePrecompile(func() {
 	ginkgo.It("tx allow list", func() {
+		if !utils.GetEnableSolidityTests() {
```

Comment:
since all of these are solidity tests can we skip all of them at once instead of one by one if `GetEnableSolidityTests` returns false?

Reply:
Now just using `--ginkgo.skip`